lib/board.rb

```diff
@@ -0,0 +1,23 @@
+class Board
+  attr_accessor :board
+
+  def initialize
+    @board = Array.new(9, ' ')
+  end
+
+  def game_board
+    puts " #{@board[0]} | #{@board[1]} | #{@board[2]} "
+    puts '-----------'
+    puts " #{@board[3]} | #{@board[4]} | #{@board[5]} "
+    puts '-----------'
+    puts " #{@board[6]} | #{@board[7]} | #{@board[8]} "
```

Comment:
puts and prints are only allowed from main.rb.
You can return your game board as a string value which you then print out from main.rb
Go back and see the previous review on this point.

Reply:
OK we made some changes to the game_board method and removed puts from the board.rb file. Thanks.